docs/lit/examples/1_Examples.jl

```diff
@@ -52,6 +52,25 @@ plot(y, ylabel=P"mass in kilograms")
 
 plot(y, ylabel="")
 
+# ### Surround
+
+# If you prefer some other formatting over the round parentheses, you can
+# supply a keyword `surround`, which can be a number of different things:
+# `:round`,`:square`,`:curly` and `:slash` do what you might expect.
+# `:slashround` and the likes will combine them. `:verbose` will insert an " in
+# units of " between the label and the unit. `nothing` will remove the
+# parentheses.  For more customizatbility, a character, a string or a `Tuple`
+# of characters or strings can be supplied, which will be inserted between the
+# label and the unit, around the unit or before, between and after, depending
+# on the length of the tuple.
```

Comment:
I think the Oxford comma would be helpful here! (Also typo on "customizatbility
".) What about something a bit shorter, like:
```
# For more customizability, a `Char`, a `String`, or a `Tuple`
# (of `Char`s or `String`s) can be supplied. They will be inserted 
# differently around the unit depending
# on the length of the tuple. See these examples:
```
And then we have more examples that show these behaviors with tuples